Assets/Scripts/GroundPound.cs

```diff
@@ -1,45 +1,30 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.InputSystem;
 
-public class GroundPound : MonoBehaviour
+public class GroundPound : Ability
 {
 
     public float dropForce =10f;
     public float stopTime = 0.5f;
     public float gravityScale = 1f;
-    private PlatformerController player;
     private Rigidbody2D body;
     private bool doGroundPound = false;
     //get animator later
 
+    protected override string InputName => "down";
```

Comment:
We now have a GroundPound input in the input system, It's currently a combination of Down and Attack. We dont have an input for just down so I can't seem to get it working